### DISCUSSION TITLE
为什么编译运行后会出现空白的页面呢

### DISCUSSION THREAD

**Discussion by Ruleigh** [whutug/whu-thesis#95] [Q&A]
我在测试使用本科毕业论文模板的时候发现每两个有字的页面直接会出现一个空白的页面，测试了硕博的论文模板也会出现空白的页面。
我检查了示例文档 whu-thesis-demo.pdf，发现其中也存在空白的页面，猜测这应该是有意为之。请问这些空白的页面意味着什么呢，是预留以待填充其他内容吗？
我是一个LaTeX新手，不太了解这方面的惯例，烦请解惑。

以下提供代码示例：
```latex
\documentclass[type=bachelor]{whu-thesis}
\begin{document}
\chapter{欢迎}
\section{Welcome to whu-thesis!}
你好，\LaTeX{}！
\end{document}
```

附件是示例代码生成的PDF文件
[Test.pdf](https://github.com/whutug/whu-thesis/files/8401635/Test.pdf)

**Comment by sikouhjw** [comment]
因为这是排版的传统[^1]。不是空白页，只是需要在奇数页开始新的一章。

[^1]: https://tex.stackexchange.com/questions/517216/why-is-openright-the-default-value-in-book

**Reply by Ruleigh** [reply]
了解了，非常感谢！
